cmake/ThirdParty.cmake

```diff
@@ -114,36 +93,16 @@ if (OPENASSETIO_ENABLE_PYTHON)
     #-------------------------------------------------------------------
     # Target to create a Python virtual environment in the install tree.
 
-    # Download core setuptools dependencies so we don't always need an
-    # online connection to local install OpenAssetIO. Need to cache the
-    # build tools explicitly due to build isolation, see:
-    # https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/#build-process
-    # This also includes runtime dependencies of packages in the
-    # resources directory, whose install will otherwise fail due to
-    # --no-index.
```

Comment:
[fix] We should add a `RELEASE_NOTES.md` call out for the no-offline aspect I think, as it is quite a significant change.

Reply:
Added a note in 59357f4